### PR TITLE
Review and integrate open95b into master

### PR DESCRIPTION
**Pull request by charlesh88**
See notes and comments in #95 for context of changes. Note that this branch incorporates *most* of the mobile changes from Shivam, but doesn't include mobile plot gestures (zoom, pan, etc.). Many other frontend tasks, such as styling Sarah Hale's work on hideable treeview and inspection, should wait until this branch is integrated into master.

**Author checklist:**
- Changes address original issue? Y
- Unit tests included and/or updated with changes? N/A
- Command line build passes? N/A
- Expect to pass code review? Y

### CODE REVIEW

**Comment by larkin** [comment]
@VWoeltjen with mobile merged, this should be clear for review & integrate.

**Comment by charlesh88** [comment]
Woo hoo! Now we just need to get open95b reviewed and merged. Thanks!

File: example/imagery/src/ImageTelemetry.js [hunk 1]
```diff
@@ -31,10 +31,11 @@ define(
...
-                "http://www.nasa.gov/393821main_Palomar_ao_bouchez_15s_after_impact_4x3_946-710.png",
-                "http://www.nasa.gov/images/content/393801main_CfhtVeillet2_4x3_516-387.jpg",
-                "http://www.nasa.gov/images/content/392790main_1024_768_GeminiNorth_NightBeforeImpact_946-710.jpg"
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/01/comet_on_16_january_2015_navcam/15205508-1-eng-GB/Comet_on_16_January_2015_NavCam.jpg",
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_7_august_a/14721226-1-eng-GB/Comet_on_7_August_a_node_full_image_2.jpg",
+                "http://http://resources0.news.com.au/images/2014/10/26/1227102/619948-e62d0d0c-5cb3-11e4-9c68-d403f5dcc36d.jpg",
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_16_august_a/14735866-1-eng-GB/Comet_on_16_August_a_node_full_image_2.jpg",
+			  	"http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/08/year_at_a_comet_may_2015/15549436-1-eng-GB/Year_at_a_comet_May_2015_node_full_image_2.jpg"
             ].map(function (url, index) {
```
Comment (VWoeltjen) [inline example/imagery/src/ImageTelemetry.js]: @charlesh88 Were these changes piggybacked intentionally or uninentionally? If we do want to keep these I'd like to remove the one that's to a non-NASA/non-ESA maintained site (strictly due to a general paranoia about checking in URLs, particularly to code that may run in the client)

**Comment by VWoeltjen** [comment]
Resolved conflicts and pushed to this branch in preparation to complete pull request. Will merge after clarification on changes to `ImageTelemetry.js`

File: example/imagery/src/ImageTelemetry.js (hunk 1 above)
Comment (larkin) [inline example/imagery/src/ImageTelemetry.js]: I believe these were intentional (as the Rosetta pictures are just beautiful), but the non-esa site should definitely go.  It's also a malformed url-- strange if it even works.

Comment (charlesh88) [inline example/imagery/src/ImageTelemetry.js]: My bad: inadvertent. I've restored the old URLs and pushed the change.
________________________________

**Comment by charlesh88** [comment]
My bad: checking those images in was inadvertent. I've restored the old URLs and pushed the change.

**Comment by VWoeltjen** [comment]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A (changes are outside scope of unit testing)
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. _Looks fantastic?_ Y

**Comment by charlesh88** [comment]
Awesome, and thanks!

**Pull request merged**